api/src/main/java/org/apache/cloudstack/api/command/user/iso/GetUploadParamsForIsoCmd.java

```diff
@@ -85,7 +85,7 @@ public Boolean isBootable() {
     }
 
     public String getDisplayText() {
-        return displayText;
+        return StringUtils.isEmpty(displayText) ? getName() : displayText;
```

Comment:
may `isBlank` be better than `isEmpty` ?

Reply:
done